website/sidebars.json

```diff
@@ -44,7 +44,7 @@
     ],
     "Controlling Fixtures": [
       "controlling-fixtures",
-      "controlling-fixtures/using-the-select-buttons-and-wheels",
```

Comment:
Was this line previously redundant?

Reply:
I have renamed the section to "changing fixture attributes" as the existing name was not really correct